src/random.jl

```diff
@@ -0,0 +1,80 @@
+#=
+Generate random NormalFormGame instances.
+=#
+
+#
+# Random Games Generating
+#
+"""
+    random_game{N}(nums_actions::NTuple{N,Int})
+
+Return a random NormalFormGame instance where the payoffs are drawn
+independently from the uniform distribution on [0, 1).
+
+# Arguements
```

Comment:
Can we leave this, until we've decided the very detail of style to follow, and then apply that to the whole docstrings?